core/pom.xml

```diff
@@ -69,6 +69,7 @@
         <dependency>
             <groupId>log4j</groupId>
```

Comment:
Still used as a test dependency in two places. Can probably switch it over to slf4j-log4j12 though. Will do so in a future PR.